bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/ProductListImpl.java

```diff
@@ -65,15 +72,15 @@
 import com.adobe.cq.commerce.magento.graphql.CategoryTree;
 import com.adobe.cq.commerce.magento.graphql.ProductInterfaceQuery;
 import com.adobe.cq.sightly.SightlyWCMMode;
+import com.adobe.granite.ui.components.ValueMapResourceWrapper;
 
-@Model(
-    adaptables = SlingHttpServletRequest.class,
-    adapters = ProductList.class,
-    resourceType = ProductListImpl.RESOURCE_TYPE)
+@Model(adaptables = SlingHttpServletRequest.class, adapters = ProductList.class, resourceType = ProductListImpl.RESOURCE_TYPE)
 public class ProductListImpl extends ProductCollectionImpl implements ProductList {
 
     public static final String RESOURCE_TYPE = "core/cif/components/commerce/productlist/v1/productlist";
     protected static final String PLACEHOLDER_DATA = "productlist-component-placeholder-data.json";
+    protected static final String PN_FRAGMENT_LOCATION = "fragmentLocation";
+    protected static final String PN_FRAGMENT_CSS_CLASS = "fragmentCssClass";
 
```

Comment:
These PN_-s would better be public like ProductList.PN_SHOW_IMAGE and NN_FRAGMENTS.